wp-content/plugins/firebase-notifications/templates/settings.php

```diff
@@ -0,0 +1,36 @@
+<?php
+ob_start();
+?>
+	<h1><?php echo get_admin_page_title(); ?></h1>
+	<div class='notice notice-success'><p><?php echo $network_settings; ?></p></div>
+	<form method='post'>
+		<?php echo wp_nonce_field( 'ig-fb-settings-nonce' ); ?>
+		<table>
+			<tr>
+				<td>Authentication Key</td><td><input type='text' name='fbn_auth_key' value='<?php echo $settings['auth_key']; ?>' size='100'></td>
+			</tr>
+			<tr>
+				<td>API URL</td><td><input type='text' name='fbn_api_url' value='<?php echo $settings['api_url']; ?>' size='100'></td>
+			</tr>
+			<tr>
+				<td><?php echo __('Groups (separate with white space)', 'firebase-notifications'); ?></td>
+				<td><input type='text' name='fbn_groups' value='".$settings['groups']."' size='100'></td>
```

Comment:
PHP tags are missing here... just a small detail.

Reply:
thanks, have not tested it yet ... thanks